src/Compilers/CSharp/Portable/Parser/Lexer.cs

```diff
@@ -2293,12 +2293,145 @@ private void LexSyntaxTrivia(bool afterFirstToken, bool isTrailing, ref SyntaxLi
                         {
                             return;
                         }
+
+                    case '>':
```

Comment:
>case [](start = 20, length = 4)

A valid program can have `>>>>>>>` at the beginning of a line. e.g.
``` c#
List<List<List<List<List<List<List<int
>>>>>>> 
x = null;
```
Are we intentionally breaking this "scenario"?